prime-router/src/test/kotlin/common/ReportNodeBuilder.kt

```diff
@@ -130,7 +142,14 @@ class ReportGraphBuilder {
             .setExternalName("test-external-name")
             .setBodyUrl(node.theReportBlobUrl)
             .setTransportResult(node.theTransportResult)
-            .setNextAction(node.reportGraphNodes.firstOrNull()?.theAction)
+            .setNextAction(
+                if (node.theNextAction != null) {
+                    node.theNextAction
+                } else {
+                    node.reportGraphNodes.firstOrNull()?.theAction
+                }
+            )
+            .setCreatedAt(graph.node.createdAt.plusMinutes(1))
```

Comment:
This was added because the nodes were generating with the same timestamp. This caused a sorting algorithm in the submission history function to fail which prevented it from finding the correct last action/report in the lineage.